LiteCore/Support/Logging.cc

```diff
@@ -34,7 +34,7 @@
 #include <android/log.h>
 #endif
 
-#if defined(__clang__) && !defined(__ANDROID__)
+#if (defined(__linux__) || defined(__APPLE__)) && !defined(__ANDROID__)
```

Comment:
I totally do not understand what's going on in this commit.  Feh.

Reply:
I was getting incorrect output on Linux here, because I realized I didn't have a path for it.  We used to use clang, but now we use GCC so `__clang__` will not be defined there.  However, the API calls between clang and GCC are identical.  I believe Android, however, does not have these calls available.  I could double check, though.